rules/TypeDeclaration/Rector/Property/PropertyTypeDeclarationRector.php

```diff
@@ -88,6 +88,10 @@ public function refactor(Node $node): ?Node
         }
 
         $phpDocInfo = $this->phpDocInfoFactory->createFromNodeOrEmpty($node);
+        
+        if ($phpDocInfo->hasInheritDoc()) {
+            return null;
+        }
```

Comment:
This introduces a new bug.

Reply:
What is it? All the tests passed?

Reply:
Yes, because the former fixture without parent class is missing.